parser-typechecker/tests/Unison/Test/TypePrinter.hs

```diff
@@ -86,6 +86,9 @@ test = scope "typeprinter" . tests $
   , tc "'(a -> b)"
   , tc "(a -> b) -> c"
   , tc "'a -> b"
+  , tc "∀ A. A -> A"
+  , tc "∀ foo.A. foo.A -> foo.A"
+  , tc "∀ A B. A -> B -> (A, B)"

```

Comment:
@atacratic and here's a test - quantifier inserted because the variables are uppercase, but we only insert one `∀` symbol.